### ISSUE TITLE
Add a way to clear localStorage in useLocalStorage

### ISSUE THREAD

**Issue by franky47** [streamich/react-use#229]
I use `useLocalStorage` a lot, and it could use a cleanup story in some cases.

Maybe as a third tuple member, have a function to call to clear the item, that respects the side-effects lifetime, as I found that simply calling `localStorage.removeItem(key)` can create a race condition with the hook recreating the value from memory.

Example:
```js
const Foo = () => {
  const [value, setValue, clearValue] = useLocalStorage('key')
  return (
    <>
      <input type="text" value={value} onChange={e => setValue(e.target.value)} />
      <button onClick={clearValue}>Clear</button>
    </>
  )
}
```

**Comment by franky47** [comment]
Here is a demo of the issue I'm having with clearing the local storage: it comes back after a page refresh.
https://codesandbox.io/s/j3xr9k25lv

**Comment by ayush987goyal** [comment]
@streamich  Hi, considering the api for `useLocalStorage` was changed recently in `v13.x`, the change in the above PR has to be done there too. @suyingtao Can you please create a PR with your changes on the latest release? :)

**Comment by franky47** [comment]
Shipped in [`v15.0.0`](https://github.com/streamich/react-use/releases/tag/v15.0.0), thanks @suyingtao  🎉
